app/charts/combo/combo-line-column-state.tsx

```diff
@@ -147,8 +149,31 @@ const useComboLineColumnState = (
         TICK_PADDING
     )
   );
+
+  const axisTitleLeft = variables.y.left.label;
+  const axisTitleRight = variables.y.right.label;
+  const axisTitleWidthLeft =
+    getTextWidth(axisTitleLeft, { fontSize: TICK_FONT_SIZE }) + TICK_PADDING;
+  const axisTitleWidthRight =
+    getTextWidth(axisTitleRight, { fontSize: TICK_FONT_SIZE }) + TICK_PADDING;
+  const overLappingTitles = axisTitleWidthLeft + axisTitleWidthRight > width;
+  const overLappingAmount = (axisTitleWidthLeft + axisTitleWidthRight) / width;
```

Comment:
Nit: I wonder if we couldn't get this in a more bullet-proof way by adding a `getTextHeight` function? Here we'd need to actually add an element to the DOM, constrain it and measure the height, as it's not really possible to do in the same way with `canvas` :/

Reply:
@bprusinowski I see, this makes total sense should we handle this in a different step or try to approach it in this PR?

Reply:
I think it makes sense to do it in this PR to keep things close to each other, I see this as a part of implementation of the wrapping behavior improvements 👍

Depending on how complex this turns out to be, there could also be a new issue. I'd investigate for ~0.5h to see if it's a huge change and make a decision based on this, if you feel like diving deeper into this :)